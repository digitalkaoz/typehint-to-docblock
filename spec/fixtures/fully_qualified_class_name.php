<?php

namespace Foo;

class Test
{
    public function does_things(\Foo $foo, \Foo\Bar\Baz $baz, \DateTime $date)
    {
    }
}
