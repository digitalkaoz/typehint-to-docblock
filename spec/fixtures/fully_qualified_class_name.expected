<?php

namespace Foo;

class Test
{
    /**
     * does_things
     *
     * @param \Foo $foo
     * @param \Foo\Bar\Baz $baz
     * @param \DateTime $date
     */
    public function does_things($foo, $baz, $date)
    {
    }
}